src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsstrengthsbasedneedsassessmentsapi/controller/AssessmentController.kt

```diff
@@ -79,4 +82,18 @@ class AssessmentController(
       assessmentVersionService.updateAnswers(it, request)
     }
   }
+
+  @RequestMapping(method = [RequestMethod.POST])
+  @ResponseStatus(HttpStatus.CREATED)
+  @Operation(description = "Create an assessment and return id")
+  @ApiResponses(
+    value = [
+      ApiResponse(responseCode = "201", description = "Assessment created"),
+    ],
+  )
+  @PreAuthorize("hasRole('ROLE_STRENGTHS_AND_NEEDS_WRITE')")
+  fun createAssessment(
+    @RequestBody
+    userDetails: UserDetails,
+  ) = assessmentService.createAndAudit(userDetails)
```

Comment:
```suggestion
  ) = assessmentService.createAndAudit(userDetails).run(CreateAssessmentResponse::from)
```

Reply:
Ah yes. Sorry - not used to returning entities to the controller. But it should be consistent! Changed

Reply:
Thanks!